analysis/backtrace/testdata/trace-specific/config.yaml

```diff
@@ -0,0 +1,25 @@
+dataflow-problems:
+  slicing:
+    - backtracepoints:
+        - target:
+            kind: "call"
+            package: "github.com/awslabs/ar-go-tools/analysis/backtrace/testdata/trace-specific"
+            method: "trackSecondArg"
+            objects:
+              - kind: "argument"
+                name: "y"
+                index: 1
+                type: "string"
+        - target:
+            kind: "call"
+            package: "github.com/awslabs/ar-go-tools/analysis/backtrace/testdata/trace-specific"
+            method: "trackSecondAndThirdArg"
+            objects:
+              - kind: "argument"
+                name: "y"
+                index: 1
+                type: "uint"
```

Comment:
There should be a test for variadic arguments to make sure this config format is flexible enough to capture them.